guides/common/modules/proc_reset-custom-ssl-certificate-to-default-self-signed-certificate-on-project.adoc

```diff
@@ -0,0 +1,27 @@
+[id="reset-custom-ssl-certificate-to-default-self-signed-certificate-on-{project-context}_{context}"]
+= Reset {customssl} certificate to default self-signed certificate on {ProjectServer}
```

Comment:
Any reason to not go with "Resetting ..." in title, anchor, and file name?

Reply:
This is an optional procedure so I assumed we may not weigh with verb in gerund. But yeah, I will stick to our style. :+1: